This is a testharness.js-based test.
Harness Error. harness_status.status = 1 , harness_status.message = Unhandled rejection: Failed to execute 'show' on 'PaymentRequest': Already called show() once
[FAIL] Calling show consumes user activation, if present
  assert_false: expected false got true
Harness: the test ran to completion.
